src/data/missions.csv

```diff
@@ -163,766 +163,766 @@
+"育成ウマ娘を才能開花で\n160回★を上げよう","Gain 160★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n165回★を上げよう","Gain 165★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n170回★を上げよう","Gain 170★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n175回★を上げよう","Gain 175★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n180回★を上げよう","Gain 180★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n185回★を上げよう","Gain 185★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n190回★を上げよう","Gain 190★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n195回★を上げよう","Gain 195★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n200回★を上げよう","Gain 200★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n205回★を上げよう","Gain 205★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n210回★を上げよう","Gain 210★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n215回★を上げよう","Gain 215★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n220回★を上げよう","Gain 220★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n225回★を上げよう","Gain 225★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n230回★を上げよう","Gain 230★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n235回★を上げよう","Gain 235★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n240回★を上げよう","Gain 240★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n245回★を上げよう","Gain 245★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n250回★を上げよう","Gain 250★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n255回★を上げよう","Gain 255★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n260回★を上げよう","Gain 260★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n265回★を上げよう","Gain 265★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n270回★を上げよう","Gain 270★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n275回★を上げよう","Gain 275★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n280回★を上げよう","Gain 280★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n285回★を上げよう","Gain 285★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n290回★を上げよう","Gain 290★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n295回★を上げよう","Gain 295★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n300回★を上げよう","Gain 300★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n305回★を上げよう","Gain 305★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n310回★を上げよう","Gain 310★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n315回★を上げよう","Gain 315★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n320回★を上げよう","Gain 320★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n325回★を上げよう","Gain 325★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n330回★を上げよう","Gain 330★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n335回★を上げよう","Gain 335★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n340回★を上げよう","Gain 340★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n345回★を上げよう","Gain 345★ through talent flowering horse girls"
+"育成ウマ娘を才能開花で\n350回★を上げよう","Gain 350★ through talent flowering horse girls"
```

Comment:
Could you change these to `talent blooming`? This is the term mostly agreed to be used in the (upcoming) wiki.  
`Gain 350★ through talent blooming`